.github/workflows/performance-tests.yml

```diff
@@ -52,11 +52,29 @@ jobs:
         run: |
           cd ../driver
           sudo make web sql="${{ secrets.DB_CONN_STRING }}"
+      - name: Check out delphi-admin
+        uses: actions/checkout@v3
+        with:
+          repository: cmu-delphi/delphi-admin
+          token: ${{ secrets.CMU_DELPHI_DEPLOY_MACHINE_PAT }}
+          path: delphi-admin
       - name: Build & run Locust
+        continue-on-error: true # sometimes ~2-5 queries fail, we shouldn't end the run if that's the case
         run: |
-          cd ../driver/repos/delphi/delphi-epidata/tests/performance
+          cd delphi-admin/load-testing/locust
           docker build -t locust .
-          docker run --net=host -v $PWD:/mnt/locust -e CSV=/mnt/locust/v4-requests-smaller.csv locust -f /mnt/locust/v4.py --host http://127.0.0.1:10080/ --users 10 --spawn-rate 1 --headless -t 15m
+          export CSV=v4-requests-small.csv
```

Comment:
question: why move the CSV filename into a variable?

(is it a stub for being able to override it in the future?)

Reply:
It's just to avoid repeating the same filename twice, since it's now used in the `wc -l` bit as well.